src/controllers/DataCreator.java

```diff
@@ -0,0 +1,174 @@
+		ArrayList<Movie> cinema1Mv = new ArrayList<Movie>();
+		ArrayList<Movie> cinema2Mv = new ArrayList<Movie>();
+		ArrayList<Movie> cinema3Mv = new ArrayList<Movie>();
+		for (int i = 0; i < 4; i++) {
+			cinema1Mv.add(movies.get(i));
+			cinema2Mv.add(movies.get(i+4));
+			cinema3Mv.add(movies.get(i+8));
+		}
+		//Create showtimes for each Cinema's Movies
+		HashMap<String, ArrayList<LocalDateTime>> cinema1St = new HashMap<String, ArrayList<LocalDateTime>>();
+		HashMap<String, ArrayList<LocalDateTime>> cinema2St = new HashMap<String, ArrayList<LocalDateTime>>();
+		HashMap<String, ArrayList<LocalDateTime>> cinema3St = new HashMap<String, ArrayList<LocalDateTime>>();
+		//Creating a fixed showtimes for all movies
+		ArrayList<LocalDateTime> showtimes = new ArrayList<LocalDateTime>(Arrays.asList(
+				LocalDateTime.parse("2022-11-14T09:30:00"), LocalDateTime.parse("2022-11-14T12:10:00"), LocalDateTime.parse("2022-11-14T15:50:00"), LocalDateTime.parse("2022-11-14T18:10:00"), LocalDateTime.parse("2022-11-14T21:00:00"),//mon
+				LocalDateTime.parse("2022-11-15T09:30:00"), LocalDateTime.parse("2022-11-15T12:10:00"), LocalDateTime.parse("2022-11-15T15:50:00"), LocalDateTime.parse("2022-11-15T18:10:00"), LocalDateTime.parse("2022-11-15T21:00:00"),//tue
+				LocalDateTime.parse("2022-11-16T09:30:00"), LocalDateTime.parse("2022-11-16T12:10:00"), LocalDateTime.parse("2022-11-16T15:50:00"), LocalDateTime.parse("2022-11-16T18:10:00"), LocalDateTime.parse("2022-11-16T21:00:00"),//wed
+				LocalDateTime.parse("2022-11-17T09:30:00"), LocalDateTime.parse("2022-11-17T12:10:00"), LocalDateTime.parse("2022-11-17T15:50:00"), LocalDateTime.parse("2022-11-17T18:10:00"), LocalDateTime.parse("2022-11-17T21:00:00"),//thur
+				LocalDateTime.parse("2022-11-18T09:30:00"), LocalDateTime.parse("2022-11-18T12:10:00"), LocalDateTime.parse("2022-11-18T15:50:00"), LocalDateTime.parse("2022-11-18T18:10:00"), LocalDateTime.parse("2022-11-18T21:00:00"),//fri
+				LocalDateTime.parse("2022-11-19T09:30:00"), LocalDateTime.parse("2022-11-19T12:10:00"), LocalDateTime.parse("2022-11-19T15:50:00"), LocalDateTime.parse("2022-11-19T18:10:00"), LocalDateTime.parse("2022-11-19T21:00:00"),//sat
+				LocalDateTime.parse("2022-11-20T09:30:00"), LocalDateTime.parse("2022-11-20T12:10:00"), LocalDateTime.parse("2022-11-20T15:50:00"), LocalDateTime.parse("2022-11-20T18:10:00"), LocalDateTime.parse("2022-11-20T21:00:00") //sun
+				));
+		//Assign the showtime for each movie
+		for (int i = 0; i < 4; i++) {
+			cinema1St.put(cinema1Mv.get(i).getTitle(), showtimes);
+			cinema1St.put(cinema2Mv.get(i).getTitle(), showtimes);
+			cinema1St.put(cinema3Mv.get(i).getTitle(), showtimes);
+		}
+		cinema.insertCinemaIntoDB("Cinema1", cinema1Mv, cinema1St);
+		cinema.insertCinemaIntoDB("Cinema2", cinema2Mv, cinema2St);
+		cinema.insertCinemaIntoDB("Cinema3", cinema3Mv, cinema3St);
+		
+		//Create CUSTOMERS
+		client.insertCustomerToDB("John", 19, "john111", "test123", "john111@gmail.com", 84324783);
```

Comment:
we need to have distinct usernames for all customers and staff. So maybe provide the customer and the staff with arbitrary names like John, Jake, and the username can just be their name in lowercase, like `john` for `John`